src/manual/Development/development-builds.md

```diff
@@ -18,7 +18,7 @@ The following are instructions for building ClamAV using CMake or Autotools with
       - [Feature Tests](#feature-tests)
   - [Building ClamAV with Autotools (v0.103 and older)](#building-clamav-with-autotools-v0103-and-older)
     - [Running autogen.sh](#running-autogensh)
-      - [Redhat / Centos / Fedora](#redhat--centos--fedora)
```

Comment:
While we don't support Redhat, the instructions for building should be identical to AlmaLinux. 

OpenSUSE will have different instructions when building from source because of different package names.  However, like Redhat/AlmaLinux/Fedora, it does support RPM's. Even so, instructions for installing our RPM will vary since they use zypper instead of dnf.